.github/CODEOWNERS

```
@@ -1,5 +1,5 @@
 # format per https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#about-code-owners
-* @colearendt @atheriel
+* @atheriel @rstudio/infraops @rstudio/ptd-dev
```

I believe it's time for me to hand this off, too. End of an era. Thank you for everything, @colearendt :saluting_face: 

```suggestion
* @rstudio/infraops @rstudio/ptd-dev
```